styleguide/media/css/pebbles/global.css

```diff
@@ -0,0 +1,144 @@
+@font-face { font-family: 'Zilla Slab'; font-weight: normal; font-style: normal; src: url("../../fonts/ZillaSlab-Regular.woff2") format("woff2"), url("../../fonts/ZillaSlab-Regular.woff") format("woff"); }
+@font-face { font-family: 'Zilla Slab'; font-weight: bold; font-style: normal; src: url("../../fonts/ZillaSlab-Bold.woff2") format("woff2"), url("../../fonts/ZillaSlab-Bold.woff") format("woff"); }
+@font-face { font-family: 'Zilla Slab'; font-weight: normal; font-style: italic; src: url("../../fonts/ZillaSlab-RegularItalic.woff2") format("woff2"), url("../../fonts/ZillaSlab-RegularItalic.woff") format("woff"); }
+@font-face { font-family: 'Zilla Slab'; font-weight: bold; font-style: italic; src: url("../../fonts/ZillaSlab-BoldItalic.woff2") format("woff2"), url("../../fonts/ZillaSlab-BoldItalic.woff") format("woff"); }
+@font-face { font-family: 'Open Sans'; font-weight: normal; font-style: normal; src: url("../../fonts/opensans-regular.woff2") format("woff2"), url("../../fonts/opensans-regular.woff") format("woff"); }
+@font-face { font-family: 'Open Sans'; font-weight: bold; font-style: normal; src: url("../../fonts/opensans-bold.woff2") format("woff2"), url("../../fonts/opensans-bold.woff") format("woff"); }
```

Comment:
This seems to be something Sass does when it compiles and I'm not sure it can be prevented easily. If you look for other instances of `url()` they're all double-quoted even though the original source is single quotes.